app/src/components/inputs/inputConditional/inputConditional.jsx

```diff
@@ -124,7 +127,7 @@ export class InputConditional extends Component {
           value={value.value}
           placeholder={placeholder}
           disabled={disabled}
-          maxLength={maxLength}
+          maxLength={inputprops.maxlength ? inputprops.maxlength : maxLength}
```

Comment:
Why not just use `maxLength` prop? What's the difference?

Reply:
maxLength is a generic prop that was set to these input fields. inputprops.maxlength is a custom property for these kind of field

Reply:
Still don't see any difference - I see 2 identical props with different names.